### DISCUSSION TITLE
What are different quarto classes for website?

### DISCUSSION THREAD

**Discussion by udurraniAtPresage** [quarto-dev/quarto-cli#9452] [Q&A] [answered]
### Description

My understanding about quarto websites is that quarto has its own classes that are based on bootstrap. For example, the boilerplate website created in rstudio with custom page layout has the class `class="quarto-container page-layout-custom page-navbar" `of the main div in the body. Where can I find more information about this? Searching for `quarto-container` on [quarto.org](http://quarto.org/) yields no results.

**Comment by mcanouil** [comment, marked as answer]
You can find that by inspecting a website or going through the codebase.
These classes are mostly used to create the website/layout.
They are not really intended to be used by most users.

**Reply by udurraniAtPresage** [reply]
Thanks.
